tests/js/lib/request.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+var fluid = require("infusion");
+
+var kettle = require("kettle");
+kettle.loadTestingSupport();
+
+fluid.defaults("gpii.express.tests.request", {
```

Comment:
See example on line 8 - the convention we have established is for "x.test" to refer to testing infrastructure common to all tests, and "x.tests" for actual test fixtures